src/main/java/hw10/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,40 @@
+package hw10;
+
+import java.io.File;
+import java.util.Objects;
+
+public class FileLoggerConfiguration {
+    private final File file;
+    private final LoggingLevel loggingLevel;
+    private final long maxFileSize;
+    private final String messageFormat;
+
+    public FileLoggerConfiguration(File file, LoggingLevel loggingLevel, long maxFileSize, String messageFormat) {
+        Objects.requireNonNull(file);
+        Objects.requireNonNull(loggingLevel);
+        Objects.requireNonNull(messageFormat);
+        if (maxFileSize < 1) {
+            throw new IllegalArgumentException("Maximum file size must be at least 1 byte.");
+        }
```

Comment:
This check can be extracted to a method, that accepts `maxFileSize` and checks it. If the check failed, then throw an exception, as you did here. If the check is passed, then simply return `maxFileSize`, so you can do it all in one line.